src/NUnitFramework/framework/Internal/PreFilter.cs

```diff
@@ -200,7 +199,7 @@ private bool MatchMethodElement(Type type)
             private bool MatchSetUpFixture(Type type)
             {
                 return IsSubNamespace(type.Namespace) &&
-                       type.GetCustomAttributes(typeof(SetUpFixtureAttribute), true).Any();
+                       type.GetCustomAttributes(typeof(SetUpFixtureAttribute), true).Length > 0;
```

Comment:
`Any` against array needs to box via `IEnumerable`, old-school is better here

Reply:
But the next dev coming in here will simplify this by moving it to Any again, 3 years down the road or some.   If old-school, it can still be added to some function, generic or standard, with a comment to explain why it is like that.

Reply:
added comment